xarray/core/formatting.py

```diff
@@ -233,9 +255,24 @@ def summarize_attr(key, value, col_width=None):
 EMPTY_REPR = u'    *empty*'
 
 
-def _calculate_col_width(mapping):
-    max_name_length = (max(len(unicode_type(k)) for k in mapping)
-                       if mapping else 0)
+def _get_col_items(mapping):
+    """Get all column items to format, including both keys of `mapping`
+    and MultiIndex levels if any.
+    """
+    col_items = []
+    for k, v in mapping.items():
+        col_items.append(k)
+        var = getattr(v, 'variable', v)
+        if isinstance(var._data, PandasIndexAdapter):
```

Comment:
I tried to import `indexVariable` in `formatting` but it gives an error. Maybe because the  `variable` module imports `common`, which in turn imports `formatting`. I don't know if there is an other (and better) way to avoid that sort of cyclic import?